resources/assets/js/review.js

```diff
@@ -137,9 +138,48 @@ $(document).ready(function() {
         $('#count-existing').html(value);
     });
 });
+// end rating review
 
+// show - hide button comment
 $(document).ready(function() {
     $("#button").click(function() {
         $("#writeComment").toggle();
     });
 });
+
+// rating comment
+var notificationsWrapper = $('.dropdown-notifications');
+var notificationsCount = $('.count-notification-circle');
+var count = parseInt($('.count-notification-circle').text());
+var pusher = new Pusher('42b42f482ce67060fc10', { cluster: 'ap1' });
+
+// Subscribe to the channel we specified in our Laravel Event
+var channel = pusher.subscribe('comment-review');
+
+// Bind a function to a Event (the full Laravel class)
+channel.bind('App\\Events\\CommentPusherEvent', function(data) {
+    var stars = '';
+    if (data.star == 1) {
```

Comment:
đoạn này mà điều kiện toàn cố định thế thì sử dụng switch case thích hợp hơn nhé
mà nhỡ data.star có ra 2.3 hoặc số lẻ ko nhỉ?
nếu mà ra số lẻ thì hiển thị ko đúng rồi

Reply:
vâng chị. vì star là số sao.Người đánh giá từ 1 đến 5 sao ạ.  tối đa đánh giá là 5 nên e để vậy